### PR TITLE
Add ability to disable user profiles at compile time.

### PR DESCRIPTION
**Pull request by chestnykh**

### CODE REVIEW

File: configure [hunk 1]
```diff
@@ -3434,8 +3438,8 @@ if test "x$enable_apparmor" = "xyes"; then :
 	HAVE_APPARMOR="-DHAVE_APPARMOR"
 
 pkg_failed=no
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for libapparmor" >&5
-$as_echo_n "checking for libapparmor... " >&6; }
+{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for AA" >&5
+$as_echo_n "checking for AA... " >&6; }
```
Comment (dseight) [inline configure:3442]: It is better to remove any unrelated changes from commit. Both this renaming and some whitespace changes.

**Review by dseight — CHANGES_REQUESTED** [review]

File: configure (hunk 1 above)
Comment (chestnykh) [reply configure]: I removed unrelated changes

**Comment by netblue30** [comment]
Thanks for the patch!
